<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarRentMain</class>
 <widget class="QMainWindow" name="CarRentMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volkswagen Financial Services</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="NameLabel">
    <property name="geometry">
     <rect>
      <x>501</x>
      <y>5</y>
      <width>259</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wellcome to Volkswagen Financial Services</string>
    </property>
   </widget>
   <widget class="QLabel" name="LoginLabel">
    <property name="geometry">
     <rect>
      <x>502</x>
      <y>34</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Login</string>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordLabel">
    <property name="geometry">
     <rect>
      <x>497</x>
      <y>70</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>  Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="SetPasswordLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>105</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string> Repeat password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_login">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>29</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>64</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_setpassword">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>99</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_car">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>441</width>
      <height>241</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="loginButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>140</y>
      <width>209</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateAccount_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>209</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Create account</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CreateAccount">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>170</y>
      <width>209</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Create account</string>
    </property>
   </widget>
   <zorder>lineEdit_setpassword</zorder>
   <zorder>lineEdit_password</zorder>
   <zorder>lineEdit_login</zorder>
   <zorder>NameLabel</zorder>
   <zorder>LoginLabel</zorder>
   <zorder>PasswordLabel</zorder>
   <zorder>SetPasswordLabel</zorder>
   <zorder>label_car</zorder>
   <zorder>loginButton</zorder>
   <zorder>CreateAccount_2</zorder>
   <zorder>CreateAccount</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_login</sender>
   <signal>textChanged(QString)</signal>
   <receiver>LoginLabel</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_password</sender>
   <signal>textChanged(QString)</signal>
   <receiver>PasswordLabel</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_setpassword</sender>
   <signal>textChanged(QString)</signal>
   <receiver>SetPasswordLabel</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>500</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
